lib/persisters/s3.js

```diff
@@ -25,7 +25,17 @@ class S3 {
    */
 
   load() {
-    return this.lambda.get(this.bucket, this.key)
+    return new Promise((resolve, reject) => {
+      this.lambda.get(this.bucket, this.key)
+        .then(resolve)
+        .catch((e) => {
+          if (String(e).indexOf('NoSuchKey') > -1) {
```

Comment:
Depends on what your thoughts are. 

This is basically to prevent a situation of where a local version does exist, but an S3 version does not, also it will prevent everything from erroring out if there is just no marker existing anywhere (eg: the first time it runs). 

Personally, I think this makes sense, jw if others agree (unfortunately Lester has unsubscribed from this repo so I couldn't tag him. :/)